### ISSUE TITLE
Question: EA/Origin games missing from list.. support for EA/Origin games possible? ability at least to import such games from windows folder?

### ISSUE THREAD

**Issue by oscarbg** [flavioislima/HeroicGamesLauncher#152]
Hi,
recently Star Wars Battlefront 2 was made free for a week on EGS..
note I can't see in Heroic library games list..
also note I wasn't expecting to be able to install directly from heroic launcher as even on Windows requires download from Origin but was expecting game to be on the list so I can use game import functionality of heroic launcher and point to the folder where I have installed on Windows..
basically was trying to see if Origin DRM will prevent such titles from working.. in any was Origin is installable on Wine Staging at least so maybe it should run if origin is launched also from same wine enviroment..
thanks..

**Comment by oscarbg** [comment]
Hi, sorry, seems first questions discused multiple times:
https://github.com/flavioislima/HeroicGamesLauncher/issues/102
but still question about ability to import such a game from a folder is interesting to hear..

**Comment by flavioislima** [comment]
A game can only be imported or installed on heroic if it has a manifest file on the epic store. Origin games doesnt have so it will never work unfortunately.

**Comment by flavioislima** [comment]
But it is planned to have an **import any game to heroic** in the future but I just don't know when. So it would be like Lutris and gamehub does right now. But I want to implement Gog before that.
